pkgs/games/gimx/env1.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/core/config_reader.c b/core/config_reader.c
```

Comment:
can you combine env1 and env2 into one patch file?

Reply:
They patch different files. Should I still combine them?

Reply:
Merged them anyway.